tools/storyteller/StoryWriter.java

```diff
@@ -50,16 +50,22 @@ void startStory(String project) {
     saveStories();
   }
 
+  void addStories(List<Story> stories){
+    if (allStories.isEmpty()) {
```

Comment:
Why only if it's empty?

Reply:
`allStories` variable was not reset during the work of the Storyteller. And when we pressed the `pause` button `allStories` did not reset.
I changed behavior in the next commit. Now, regardless of whether Storyteller was closed completely or the pause button was pressed `allStories` is reset.